desafio-02/zezola/java/main.java

```diff
@@ -0,0 +1,21 @@
+public class NumerosPrimos {
+    public static void main(String[] args) {
+
+        for (int i = 2; i < 10000; i++) {
+            if (primeChecker(i)) {
+                System.out.println(i);
+            }
+        }
+
+    }
+
+    public static boolean primeChecker(int num) {
```

Comment:
Como esta função é acessada apenas dentro da classe, não seria melhor deixá-la como `private` ao invés de `public`?

Reply:
Além disso, uma dica que sempre gosto de utilizar na hora de nomear funções cujo retorno é do tipo `boolean` é utilizar predicados que levem a resposta de sim/não(ou verdade/falso), como `isEmpty`, `isNotPrime`, `isEven`, `containsElement` e etc.

Como referência, indico [este pequeno trecho com exemplos do StackExchange](https://www.bigfont.ca/how-to-name-boolean-returning-functions-and-methods/)

Reply:
Obrigado pelo feedback!